testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/suite/suites/Suite2.java

```diff
@@ -38,16 +38,16 @@ public List<SuiteTestRun> getTestRuns(EnvironmentConfig config)
                         .withGroups("configured_features", "hdfs_no_impersonation")
                         .build(),
                 testOnEnvironment(EnvSinglenodeKerberosHdfsNoImpersonation.class)
-                        .withGroups("configured_features", "storage_formats", "hdfs_no_impersonation")
+                        .withGroups("configured_features", "storage_formats", "hdfs_no_impersonation", "hive_kerberos")
                         .build(),
                 testOnEnvironment(EnvSinglenodeKerberosHiveNoImpersonationWithCredentialCache.class)
-                        .withGroups("configured_features", "storage_formats", "hdfs_no_impersonation")
+                        .withGroups("configured_features", "storage_formats", "hdfs_no_impersonation", "hive_kerberos")
```

Comment:
Do we need to run this test on kerberos with credentialCache environment

Reply:
Probably not since the cache's ticket lifetime is larger than the time the test waits (30m vs 30seconds).

I'll remove.

Reply:
removed from credential cache tests